stdpopsim/catalog/CanFam/genome_data.py

```diff
@@ -1,47 +1,47 @@
 # File autogenerated from Ensembl REST API. Do not edit.
 data = {
-    "assembly_accession": "GCA_000002285.2",
-    "assembly_name": "CanFam3.1",
+    "assembly_accession": "GCA_014441545.1",
+    "assembly_name": "ROS_Cfam_1.0",
     "chromosomes": {
-        "1": {"length": 122678785, "synonyms": []},
-        "2": {"length": 85426708, "synonyms": []},
-        "3": {"length": 91889043, "synonyms": []},
-        "4": {"length": 88276631, "synonyms": []},
-        "5": {"length": 88915250, "synonyms": []},
-        "6": {"length": 77573801, "synonyms": []},
-        "7": {"length": 80974532, "synonyms": []},
-        "8": {"length": 74330416, "synonyms": []},
-        "9": {"length": 61074082, "synonyms": []},
-        "10": {"length": 69331447, "synonyms": []},
-        "11": {"length": 74389097, "synonyms": []},
-        "12": {"length": 72498081, "synonyms": []},
-        "13": {"length": 63241923, "synonyms": []},
-        "14": {"length": 60966679, "synonyms": []},
-        "15": {"length": 64190966, "synonyms": []},
-        "16": {"length": 59632846, "synonyms": []},
-        "17": {"length": 64289059, "synonyms": []},
-        "18": {"length": 55844845, "synonyms": []},
-        "19": {"length": 53741614, "synonyms": []},
-        "20": {"length": 58134056, "synonyms": []},
-        "21": {"length": 50858623, "synonyms": []},
-        "22": {"length": 61439934, "synonyms": []},
-        "23": {"length": 52294480, "synonyms": []},
-        "24": {"length": 47698779, "synonyms": []},
-        "25": {"length": 51628933, "synonyms": []},
-        "26": {"length": 38964690, "synonyms": []},
-        "27": {"length": 45876710, "synonyms": []},
-        "28": {"length": 41182112, "synonyms": []},
-        "29": {"length": 41845238, "synonyms": []},
-        "30": {"length": 40214260, "synonyms": []},
-        "31": {"length": 39895921, "synonyms": []},
-        "32": {"length": 38810281, "synonyms": []},
-        "33": {"length": 31377067, "synonyms": []},
-        "34": {"length": 42124431, "synonyms": []},
-        "35": {"length": 26524999, "synonyms": []},
-        "36": {"length": 30810995, "synonyms": []},
-        "37": {"length": 30902991, "synonyms": []},
-        "38": {"length": 23914537, "synonyms": []},
-        "X": {"length": 123869142, "synonyms": []},
-        "MT": {"length": 16727, "synonyms": []},
+        "1": {"length": 123313939, "synonyms": []},
```

Comment:
Looks like the chromosome lengths changed between these two builds of CanFam, and new build doesn't have a mitochondrial genome. We probably want to stick with old version to avoid conflicts with recombination map.

Reply:
Hm - unlike in other species, it appears to legit [not have MT](http://rest.ensembl.org/info/assembly/canis_lupus_familiaris?synonyms=1) any more.